### ISSUE TITLE
ignore_index in the cross entropy loss

### ISSUE THREAD

**Issue by ttumyche** [lucidrains/parti-pytorch#8]
Thank you for taking the time to implement Parti.

I wonder why the ignore_index of the loss function is set to zero.
Zero is also included in the codebook index, and the input image can be converted to the sequence of discrete tokens with a zero index.

https://github.com/lucidrains/parti-pytorch/blob/5a93f94161007d6ea2737cc45dc907fe8fc46138/parti_pytorch/parti_pytorch.py#L375

**Comment by lucidrains** [comment]
@ttumyche yes you are right, i think this is an error on my part

**Comment by ttumyche** [comment]
Thank you for the reply :)
